test/integration/full/html-has-lang/html-has-lang-pass.html

```diff
@@ -16,10 +17,15 @@
 		var assert = chai.assert;
 	</script>
 </head>
+
 <body>
-	<iframe id="frame1" src="frames/level1.html"></iframe>
+	<!-- 
+			Note: 
+			This rule does not include `iframe` uses matches "window-is-top.js" 
```

Comment:
Yeah, but you do actually need to test that. Don't remove the frames from the examples.